### ISSUE TITLE
[Rogue] Rogue Stuns (Cheap Shot/Kidney Shot) not labeled as Physical Spells

### ISSUE THREAD

**Issue by bluemanguy112** [Atlantiss/NetherwingBugtracker#8533]
**Description**: 
See description from enclosed screenshot below of a chat in the TBC Research Discord established for TBCC between players and Blizzard Devs.

**Current behaviour**:
Rogue Stuns currently completely ignore this and do not require Spellhit to mitigate the innate 4% Resist Chance. 

https://cdn.discordapp.com/attachments/1063133316909371522/1125497634350702662/image.png

**Comment by Wolffenstein** [comment]
Fixed in rev 3823.

**Comment by sasadubi** [comment]
@Wolffenstein  So what's the conclusion here? Rogue need spell hit now on Stormforge?
Can you pls shed some lights how the WoW TBC works in code. Is there a difference between Spells and Abilities? And why are Stuns considered to be Spells now?
I feel we really need some word and reasoning behind this change.
I'm playing private TBC servers since Feenix Archangel 2008, and I've never heard of this that a rogue abilities are actually counted as spells. And that a Rogue has "Double approach to Hit rating", aka needing physical hit rating cap, and now a spell hit cap.

**Comment by Godmatik** [comment]
Wolf said on discord he is reverting this for now until there is further proof given

**Comment by sasadubi** [comment]
@Godmatik  yeah saw it, ty man.

**Comment by bluemanguy112** [comment]
Does this mean all physical spells (i.e. intercept, piercing howl, freezing trap, etc) now do not require spellhit either? The point I was making and always tried to make is that it is unfair for only one class - Rogue - to not be affected by this (Blizzlike-)Mechanic. Either keep it for all incl. Rogues or remove it for everyone.

Edit: added a screenshot from the same discord of a known rogue player acknowledging the functionality since I don't want to dox Blizzard Devs.
![image](https://github.com/Atlantiss/NetherwingBugtracker/assets/62884794/f1345d09-3300-4e44-94f1-f3f5c334ee40)
 

Would appreciate some clarification.

P.S. the "it was never like this on other Pservers before" doesn't hold up since the functionality was discovered during TBC classic.

**Comment by sasadubi** [comment]
@bluemanguy112
Few things I want to add as response to your comment, and please, this is not bitching or ranting, I just wanted to provide you my reasoning behind my comment above. :)

1. The stun resists that were occurring after the "innate stun fix" were huge. Resto druids and Rogues without meta gem resisting straight CS resist into KS resist. Not to mention Disci priests or Orcs, it was just unreal, because on top of racial and talents it added this 3-4% of chance to resist a spell.

2. TBC Classic itself is not credible enough, because TBC 2007 and TBC Classic were different games. Example Given:
a) eg: Cloak of Shadows was always 90% resist chance while it's on, I played TBC in 2007 and played on 7-8 private TBC servers before TBC Classic. Blizzard comes and says in PVP, Cloak is actually 80%, and 90% goes against NPC spells. For whole season 1, rogues were getting dots through Cloak 24/7. Blizzard steps out later in s2 and says it should be 90% and fixes it.
b) Mage Ice Lance into Frost Nova. On TBC Classic PTR people noticed that Nova simply does not break and mage could do 6-7 Ice Lances into single Nova without any problems. Blizzard Dev's respond that's how it was in 2007, and people countered them with ton of YouTube proofs in form of old PVP movies/highlights from good players. Then in s2, Blizzard against admits it was a mistake and nerfs it multiple times until like in s3 it was back to normal.

3. I'm 100% certain that in 2007, as a rogue I can walk out of flare, and after I'm gone from animation of flare, after 1sec like, I can vanish. On TBC Classic, I legit sprinted out of flare for 20-30 yards, and still had Flare debuff for 6-7 sec.

4. There will be always Left and Right. People claiming that innate stun resist chance was a thing and people like me claiming it wasn't a thing. I never gemmed spell hit for 15 years of TBC.
----------------------------------------------------------------------------------
There are many more scenarios in which I think TBC Classic generally failed it's customers, and everyone thinks private servers were simply better.

TLDR 1: Even if we find blue post or something from 2007 or TBC Classic, we need to question the validity of it. All I'm saying, I never had for example, Resto druid, or (non-orc) Rogue without 5% meta resisting my stun. A Rogue can definitely front-face and dodge/parry it, but not resist.

@bluemanguy112  as for your concern for other abilities, someone else can jump in and add shed some light.

Peace mate.

**Comment by bluemanguy112** [comment]
Since you felt compelled to bring flare up for some reason btw: https://github.com/Aeded/BCC_Buglist/issues/5
